commands/playerdata.js

```diff
@@ -0,0 +1,54 @@
+//player data storage access
+
+module.exports = {
+    name: 'playerdata',
+    aka: ['pd', 'userdata'],
+    description: 'Get stats (datastore info) for any user (Board+)',
+    guildOnly: true,
+    args: true,
+    helpLevel: 'staff',
+    required_role: role.board,
+    usage: ` <nameToLookup>`,
+    execute(msg, args, _, internal_error) {
+        let name = args[0];
+                
+        //this command unlike its brother pd2 DOES require a name to be specified. You can still look up yourself but you must add in your username to look up.
+        if (!name) 
+        {
+            msg.channel.send(`Please pick a name first. Just the name - CAPS COUNT (use \`playerdata2\` if you dont want to add in the name`)
+                .catch((err) => { internal_error(err); return });
+            return;
+        }
+        if (name) 
+        {
+            const fs  = require('fs');
+            let rawdata = fs.readFileSync('/home/exp_admin/api_v2/persistent_storage.json');
+            let mydata = JSON.parse(rawdata);
+            key1 = `${name}`; // in dataFile
+            let checkdata = mydata["PlayerData"][key1];
+            let finaldata;
+            
+            if(mydata["PlayerData"][key1] )
+            {
+                finaldata = mydata["PlayerData"][key1]["Statistics"];
+                msg.channel.send(`${name}:\n\`\`\`json\n${JSON.stringify(finaldata, null, 2)}\`\`\``);
```

Comment:
you need to make sure this cant be more then 2000 chars just to be safe

Reply:
```suggestion
                msg.channel.send(`${name}:\n\`\`\`json\n${JSON.stringify(finaldata, null, 2)}\`\`\``);
```
This should work to split it automatically. Though I will need to test it and then append the ```

Reply:
no as then it wont have the ``` if its more then 2000

Reply:
will have to set up a check later, We should make an issue for this one then.

Reply:
Will have to fix in new issue.